backend/src/Database.hs

```diff
@@ -64,10 +72,11 @@ getPlayerByName :: PlayerName -> AppM (Maybe ReadPlayer)
 getPlayerByName name = do
   env <- ask
   liftIO . withResource env.dbPool $ \conn -> do
-    player <- query conn "SELECT * FROM Players WHERE name=(?)" (Only name)
-    case player of
-      [p] -> pure (Just p)
-      _ -> pure Nothing
+    result <- query conn "SELECT * FROM Players WHERE name = ?" (Only name)
```

Comment:
What does changing `name=(?)` to `name = ?` do? Rest of the change makes sense ✔️

Reply:
No difference, the parens are just optional there, so I took 'em out :)

Reply:
💯